src/convertToObject.js

```diff
@@ -6,7 +6,36 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStyle = sourceString.split(';');
+  const style = {};
+
+  Object.create(style);
+
+  sourceStyle.forEach((source) => {
+    const data = source.split(':');
+
+    if (data[0].trim()) {
+      const key = data[0].trim();
+      const value = data[1];
+
+      const s = value;
+      let result;
```

Comment:
The variable `s` and `result` are unnecessary. You can directly use `value` after trimming it.